### PR TITLE
Remove old contracts and implementation supporting them

### PR DESCRIPTION
**Pull request by 0xmovses**
# Summary
- **RFCs**: [Link to RFC](#./link/to/rfc), [Link to RFC](#./link/to/rfc), or $\emptyset$.
- **Categories**: any of `protocol-units`, `networks`, `scripts`, `util`, `cicd`, or `misc`.

# Changelog

# Testing

# Outstanding issues

### CODE REVIEW

**Review by musitdev — COMMENTED** [review]
You don't update the bridge code to not use the contract?

**Comment by 0xmovses** [comment]
Still working on this

File: protocol-units/bridge/service/src/chains/movement/client_framework.rs [hunk 1]
```diff
@@ -66,7 +62,7 @@ impl MovementClientFramework {
 		let native_address = AccountAddress::from_hex_literal(&config.movement_native_address)?;
 		Ok(MovementClientFramework {
 			native_address,
-			non_native_address: Vec::new(), //dummy for now
+			non_native_address: Vec::new(), //This seems unused
```
Comment (musitdev) [inline protocol-units/bridge/service/src/chains/movement/client_framework.rs:65]: Yes you can remove. Not used with the framework.

**Review by musitdev — APPROVED** [review]
No remarks, you can remove the `non_native_address` that is not used.

**Review by andygolay — APPROVED** [review]
It builds, still need to fix E2E tests but this looks good to me.

**Pull request merged**